<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>perf_graph</class>
 <widget class="QDialog" name="perf_graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Performance</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(235, 235, 235);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>720</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="close_pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="reset_pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Reset All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="fps_lcdNumber">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="fps_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Images/Sec:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="images_lcdNumber">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>470</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="digitCount">
    <number>14</number>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="images_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Img Processed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="header_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 42pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Performance</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label_Radeon">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/Radeon_Instinct.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label_AMD">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/AMD_NEW_logo_150.png</pixmap>
   </property>
  </widget>
  <widget class="QLCDNumber" name="maxFPS_lcdNumber">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="maxFPS_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 28pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>MAX FPS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Model_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
  <widget class="QLabel" name="modelName_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="NGPU_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>GPU:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="GPU_lcdNumber">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="startName_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Start Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="StartTime_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ElapsedTimeName_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>610</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>Elapsed Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="elapsedTime_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>610</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="NCPU_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>CPU:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="CPU_lcdNumber">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="GPUName_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CPUName_label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>290</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 75 24pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
